integration-cli/docker_api_inspect_unix_test.go

```diff
@@ -16,7 +16,7 @@ func (s *DockerSuite) TestInspectApiCpusetInConfigPre120(c *check.C) {
 	testRequires(c, cgroupCpuset)
 
 	name := "cpusetinconfig-pre120"
-	dockerCmd(c, "run", "--name", name, "--cpuset", "0-1", "busybox", "true")
+	dockerCmd(c, "run", "--name", name, "--cpuset-cpus", "0-1", "busybox", "true")
```

Comment:
since there wasn't a `-cpuset` in the parse file for docker run I think it was just mistaken and it was `--cpuset` to be removed